lambda_function/aws/resources/acm_cert_handler.py

```diff
@@ -0,0 +1,80 @@
+import json
+import logging
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+import boto3
+import consts
+from aws.resources.base_handler import BaseHandler
+from port.entities import create_entities_json, handle_entities
+
+logger = logging.getLogger(__name__)
+
+
+class ACMHandler(BaseHandler):
+    def handle(self):
+        for region in list(self.regions):
+            aws_acm_client = boto3.client("acm", region_name=region)
+            try:
```

Comment:
You should follow the [pattern](https://github.com/port-labs/port-aws-exporter/blob/2bb2a6d1a4cfc3270df896193840b54d4e939d80/lambda_function/aws/resources/cloudformation_handler.py#L20) of using `NextToken` instead of using a paginator, because when the lambda is close to timeout and launch another instance it should continue from the last handled page. Moreover, make sure to call `self._cleanup_regions(region)` at the end of the loop, from the same reasons.

Reply:
Also add a log for list with the value of the current region, like in the other handlers.